apps/back/migrations/Version20230905094943.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230905094943 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('CREATE TABLE course_user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, username VARCHAR(180) NOT NULL, views_count INT DEFAULT NULL, last_view INT DEFAULT NULL, UNIQUE INDEX UNIQ_45310B4FE7927C74 (email), UNIQUE INDEX UNIQ_45310B4FF85E0677 (username), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```

Comment:
We prefer use uuid instead of id.

In your constructor, you will juste have uuid from symfony and modified this migrations
`id CHAR(36) NOT NULL COMMENT \'(DC2Type:uuid)\'`